pylint/test/functional/arithmetic_operators.py

```diff
@@ -0,0 +1,8 @@
+# pylint: disable=missing-docstring,invalid-name,unused-argument
+def sample_function(*args, **kwargs):
```

Comment:
I think this could be made to work with some tweaking.  I think this makes sense in the cases where both operands are not `BinOp`s and neither is the parent (with the exception of being the RHS in a `AssignTypeMixin`).  This would catch `x = a*4` but allow `a = a*4 + 2`.  What do you think @PCManticore @The-Compiler ?